garage/misc/krylov.py

```diff
@@ -3,24 +3,24 @@
 EPS = np.finfo('float64').tiny
 
 
-def cg(f_Ax, b, cg_iters=10, callback=None, verbose=False, residual_tol=1e-10):
-    """
-    Demmel p 312
-    """
+def cg(f_ax, b, cg_iters=10, callback=None, verbose=False, residual_tol=1e-10):
```

Comment:
perhaps just `# noqa: error-code` this file for now, since it seems like the capitalization are important for meaning in this algorithm (they correspond to notation from equations)